xtask/src/install.rs

```diff
@@ -74,17 +109,13 @@ fn fix_path_for_mac() -> Result<()> {
     Ok(())
 }
 
-fn install_client(ClientOpt::VsCode: ClientOpt) -> Result<()> {
-    let _dir = pushd("./editors/code")?;
+fn install_client(client_opt: ClientOpt) -> Result<()> {
+    let _dir = pushd("./editors/code");
 
     let find_code = |f: fn(&str) -> bool| -> Result<&'static str> {
-        ["code", "code-insiders", "codium", "code-oss"]
-            .iter()
-            .copied()
-            .find(|bin| f(bin))
-            .ok_or_else(|| {
-                format_err!("Can't execute `code --version`. Perhaps it is not in $PATH?")
-            })
+        client_opt.as_cmds().iter().copied().find(|bin| f(bin)).ok_or_else(|| {
+            format_err!("Can't execute `code --version`. Perhaps it is not in $PATH?")
```

Comment:
When using `ClientOpt::All` it should attempt to install for all found vscode variants I think. Or it should be renamed to `ClientOpt::Any`.

Reply:
That is not what xtask install --client-code does today. I think that's outside the scope of this PR.

I'll change the name though to Any